commands/sovde.py

```diff
@@ -5,47 +5,52 @@
-        await interaction.response.send_message(content="[Sovdee's Variable Tutorial](https://sovdee.gitbook.io/skript-tutorials/core-concepts/variables)", view=DeleteButton(interaction.user.id))
+        await interaction.response.send_message(content="[Sovde's Variable Tutorial](https://sovdee.gitbook.io/skript-tutorials/core-concepts/variables)", view=DeleteButton(interaction.user.id))
 
-    @app_commands.command(description="Home page of Sovdee's tutorials")
+    @app_commands.command(description="Home page of Sovde's skript tutorials")
     async def home(self, interaction: discord.Interaction) -> None:
         await interaction.response.send_message(content="https://sovdee.gitbook.io/skript-tutorials/", view=DeleteButton(interaction.user.id))
 
     @app_commands.command(name="indentation", description="Indentation and Program Flow")
     async def indent(self, interaction: discord.Interaction) -> None:
-        await interaction.response.send_message(content="[Sovdee's Indentation Tutorial]("
+        await interaction.response.send_message(content="[Sovde's Indentation Tutorial]("
                                                         "https://sovdee.gitbook.io/skript-tutorials/core-concepts/indentation)", view=DeleteButton(interaction.user.id))
 
     @app_commands.command(name="commands", description="Custom Commands")
     async def commands(self, interaction: discord.Interaction) -> None:
-        await interaction.response.send_message(content="[Sovdee's Custom Command Tutorial]("
+        await interaction.response.send_message(content="[Sovde's Custom Command Tutorial]("
                                                         "https://sovdee.gitbook.io/skript-tutorials/core-concepts/commands)", view=DeleteButton(interaction.user.id))
 
     @app_commands.command(name="vectors", description="Vector tutorials")
     async def vectors(self, interaction: discord.Interaction) -> None:
-        await interaction.response.send_message(content="[Sovdee's Vector Tutorial]("
+        await interaction.response.send_message(content="[Sovde's Vector Tutorial]("
                                                         "https://sovdee.gitbook.io/skript-tutorials/core-concepts/vectors)", view=DeleteButton(interaction.user.id))
 
     @app_commands.command(name="functions", description="Functions tutorial")
     async def functions(self, interaction: discord.Interaction) -> None:
-        await interaction.response.send_message(content="[Sovdee's Function Tutorial]("
+        await interaction.response.send_message(content="[Sovde's Function Tutorial]("
                                                         "https://sovdee.gitbook.io/skript-tutorials/core-concepts/indentation/functions)", view=DeleteButton(interaction.user.id))
 
+    @app_commands.command(name="list-basics", description="Variable List Basics Tutorial")
+    async def functions(self, interaction: discord.Interaction) -> None:
```

Comment:
This function shouldn't be called functions, it should be called list_basics
```suggestion
    async def list_basics(self, interaction: discord.Interaction) -> None:
```